index.html

```diff
@@ -1,1982 +1,939 @@
+                            alt="social-media-icons"></a></li>
+                <li><a href="https://twitter.com/topragagiden"><img src="./icons/twitter.png"
+                            alt="social-media-icons"></a>
+                </li>
+            </ul>
+            <svg width="42" height="42" viewBox="0 0 42 42" fill="none" xmlns="http://www.w3.org/2000/svg">
+                <rect x="1" y="1" width="40" height="40" rx="20" fill="white" />
+                <path fill-rule="evenodd" clip-rule="evenodd"
+                    d="M14.2929 17.2929C14.6834 16.9024 15.3166 16.9024 15.7071 17.2929L21 22.5858L26.2929 17.2929C26.6834 16.9024 27.3166 16.9024 27.7071 17.2929C28.0976 17.6834 28.0976 18.3166 27.7071 18.7071L21.7071 24.7071C21.3166 25.0976 20.6834 25.0976 20.2929 24.7071L14.2929 18.7071C13.9024 18.3166 13.9024 17.6834 14.2929 17.2929Z"
+                    fill="#172B4D" />
+                <rect x="1" y="1" width="40" height="40" rx="20" stroke="#DFE1E6" />
+            </svg>
+        </div>
+    </section>
+    <section id="my-works">
+        <div>
+            <h1>My Recent Works</h1>
+            <svg width="49" height="4" viewBox="0 0 49 4" fill="none" xmlns="http://www.w3.org/2000/svg">
+                <rect x="0.5" width="48" height="4" rx="2" fill="#36B37E" />
+            </svg>
+        </div>
+        <div class="card-container">
+            <div class="card">
+                <img alt="screenhost of my work" src="#">
+                <h1></h1>
+                <ul class="tech">
+                </ul>
+                <a class="btn-green" href="#">See Project</a>
+            </div>
+            <div class="card">
+                <img alt="screenhost of my work" src="#">
+                <h1></h1>
+                <ul class="tech">
+
+                </ul>
+                <a class="btn-green" href="#">See Project</a>
+            </div>
+            <div class="card">
+                <img alt="screenhost of my work" src="#">
+                <h1></h1>
+                <ul class="tech">
+                </ul>
+                <a class="btn-green" href="#">See Project</a>
+            </div>
+            <div class="card">
+                <img alt="screenhost of my work" src="#">
+                <h1></h1>
+                <ul class="tech">
+
+                </ul>
+                <a class="btn-green" href="#">See Project</a>
+            </div>
+            <div class="card">
+                <img alt="screenhost of my work" src="#">
+                <h1></h1>
+                <ul class="tech">
+
+                </ul>
+                <a class="btn-green" href="#">See Project</a>
+            </div>
+            <div class="card">
+                <img alt="screenhost of my work" src="#">
+                <h1></h1>
+                <ul class="tech">
+
+                </ul>
+                <a class="btn-green" href="#">See Project</a>
+            </div>
```

Comment:
## BEAUTIFUL ANIMATIONS 👏
- [ ] Your popups are working well as stored in the array of information of all projects 😀. Please also create all of these cards using javascript, in which you have changed the image, heading, and technologies 👍. All the code should be removed and created using javascript 😎. Keep on creating such an awesome project with cool animations 💻. GOOD LUCK 🍀! 

**Screenshot of the marked code** 

![image](https://user-images.githubusercontent.com/69011963/151345593-4b5af2ec-c166-44ef-abcd-f79111076544.png)